### ISSUE TITLE
Setting font size/family for text theme

### ISSUE THREAD

**Issue by daniel-v** [rxlabz/panache#15]
Missing the ability to set the font size and the family.

Did I look something over or are these actually a missing features? In which case this issue is a feature request :)

**Comment by rxlabz** [comment]
Yes, I removed the font size fields because changing size caused crash in browser #14 I'll try to add it back ASAP

**Comment by daniel-v** [comment]
Ohh, nice! Thank you
